### DISCUSSION TITLE
[V3] Plugin timeGrid doesn't works when not alone

### DISCUSSION THREAD

**Discussion by swrtom** [saade/filament-fullcalendar#121] [General]
Hello,
This code show timeGrid but when I add "dayGrid" in plugins array, it is not showing. Also, there is no buttons to switch from day, week, month in timeGrid view see top right empty button. I think I missed something about configuration to add timeGrid

```
->plugin(
                FilamentFullCalendarPlugin::make()
                    ->schedulerLicenseKey('CC-Attribution-NonCommercial-NoDerivatives')
                    ->selectable()
                    ->editable()
                    ->plugins(['timeGrid'], false)
                    ->config([])
```

TimeGrid view in my app :

![image](https://github.com/saade/filament-fullcalendar/assets/44288315/94606e6f-c298-4445-8e0a-0df02b40d745)

Thank you for help !

**Comment by swrtom** [comment]
Solved ! I added this into config array : 
```
                    ->config([
                        'headerToolbar' => [
                            'right' => 'dayGridMonth,timeGridWeek,timeGridDay',
                        ],
                    ])
```

**Reply by KonstantinosVazaios** [reply]
hey @sheepbild does the time grid feature of fullCallendar work on Fillament FullCalendar? How did you set it up?
